pkg/appconsts/initial_consts.go

```diff
@@ -22,6 +22,13 @@ const (
 	// a nodes `CheckTx` and thus not be proposed by that node.
 	DefaultMinGasPrice = 0.002
 
+	// GlobalMinGasPrice is used in the processProposal to ensure
+	// that all transactions have a gas price greater than or equal to this value.
+	GlobalMinGasPrice = DefaultMinGasPrice
```

Comment:
This should be viewed as a temporary home for now as we will want to either make it a governance parameter or have some other mechanism which picks this value. I don't think v2 will simply have this hardcoded value.

If we want to move it, we can shift it to the v2 directory. I'm easy either way

Reply:
Sounds good, I like `v2/app_consts.go`. 

I think I'd rather it _not_ be a governance modifiable param but I think we can discuss that as a follow up. Mostly b/c I don't think we should have any governance params long term and instead we should modify params via hard-forks after reaching rough consensus off-chain. So eventually we would move from governance params to versioned params.

Reply:
Since this PR already has two approvals, I don't think we need to block on just this constant move. We can move the constant in a follow up PR if that's easier

Reply:
I'll make this change in a follow-up pr @rootulp @cmwaters